Tensile/Source/client/main.cpp

```diff
@@ -467,7 +467,7 @@ int main(int argc, const char* argv[])
     listeners.addListener(std::make_shared<ProgressListener>());
 
     listeners.addListener(std::make_shared<BenchmarkTimer>(args, *hardware));
-    listeners.addListener(std::make_shared<HardwareMonitorListener>(args));
+    //listeners.addListener(std::make_shared<HardwareMonitorListener>(args));
```

Comment:
Global parameter `HardwareMonitor=False` will disable this if you have problems.